src/os/data/histo/colormodel.js

```diff
@@ -176,6 +176,36 @@ os.data.histo.ColorModel.prototype.getResults = function() {
 };
 
 
+/**
+ * Get the bin/color pairs currently applied by this color model.
+ *
+ * @return {Object<string, string>}
+ * @suppress {accessControls, checkTypes} To allow direct access to bin data.
```

Comment:
Do you need the `checkTypes` suppression here? We really don't like to use that one unless absolutely required.